src/features/admin/components/CouriersOrderCard/CouriersOrderCard.jsx

```diff
@@ -16,8 +23,7 @@ function CouriersOrderCard({ order, setActive }) {
 
         <ul className="order-card__list">
           <li>
-            Restaurant: <strong>{order.restaurant.name}</strong>,{' '}
-            {order.restaurant.address}
+            Restaurant: <strong>{restaurant.name}</strong>, {restaurant.address}
```

Comment:
![image](https://user-images.githubusercontent.com/38314197/221177039-1444c61b-5a93-4c9b-8709-79b52364f059.png)

If there is no restaurant found, this will produce error.